packages/react/src/components/SidePanel/SidePanel.jsx

```diff
@@ -52,6 +55,8 @@ const propTypes = {
     primaryButtonLabel: PropTypes.string,
     secondaryButtonLabel: PropTypes.string,
   }),
+  /** Toggle side panel busy state */
+  busy: PropTypes.bool,
```

Comment:
Can we change to isBusy to follow convention?
```suggestion
  isBusy: PropTypes.bool,
```

Reply:
> Can we change to isBusy to follow convention?

Updated.